sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/plans/logical/basicOperators.scala

```diff
@@ -512,24 +512,35 @@ private[sql] object Expand {
  * a input row.
  *
  * @param projections to apply
- * @param output of all projections.
  * @param child operator.
  */
 case class Expand(
-    projections: Seq[Seq[Expression]],
-    output: Seq[Attribute],
+    projections: Seq[Seq[NamedExpression]],
     child: LogicalPlan) extends UnaryNode {
+  override def output: Seq[Attribute] = {
+    // Take the first projection as output
+    val preOutput = projections.head.map(_.toAttribute)
```

Comment:
yea, kind of. If we only make the first projection as `Seq[NamedExpression]`, I think it might cause little confusing.